website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -89,10 +149,57 @@ The content type property is only relevant within Figma and doesn’t exist as a
 
 The `content` supports any custom content, local components, or Helios components via an `instance swap` property (customInstance) in Figma. In code, `yield` is supported.
 
-### Default
+#### Default
 
 ![Example of content in accordion item](/assets/components/accordion/content-text.png =690x*)
 
-### Custom
+#### Custom
+
+![Example of content in accordion item](/assets/components/accordion/content-custom.png =690x*)
+
+### Nesting Accordions
+
+!!! Info
+
+Nesting Accordions is only supported in code, as Figma does not support nesting an instance inside itself. While you can work around this by detaching the HDS component and turning it into a local component, then inserting the local component within the HDS component, we do not recommend this approach. Doing so means your local Accordion component will not receive future updates. It's best to handle nested Accordions in code to maintain update compatibility. [Link to Code example]
+!!!
+
+Nesting Accordions can help organize complex content but should be used in moderation. This feature is only recommended for the `flush` variant.
+
+!!! Do
+
+Use nested Accordions for up to two layers of related content. You can nest `flush` Accordions within each other or inside `card` type Accordions. 
+
+![Image example, card and flush and flush and flush](/assets/components/accordion/accordion-nesting-example.png)
+!!!
+
+!!! Dont
+
+Avoid nesting `card` Accordions within each other to prevent excessive visual noise.
+
+![Show images of two card variants nested in each other](/assets/components/accordion/accordion-nesting-example-dont.png)
```

Comment:
[nit]

```suggestion
![Image of two card variants nested in each other](/assets/components/accordion/accordion-nesting-example-dont.png)
```

Reply:
Avoid using "image of" language in alt text.

Reply:
Change alt text to
=>
Two card variants nested within each other